src/sif/pedestriancost.cc

```diff
@@ -89,6 +89,8 @@ const std::string kDefaultPedestrianType = "foot";
 // hills) to 1 (take hills when they offer a more direct, less time, path).
 constexpr float kDefaultUseHills = 0.5f;
 
+constexpr float kDefaultUseLit = 0.f;
```

Comment:
@chrstnbwnkl same comment as above.  Why not 0.5f for the default.

Reply:
@chrstnbwnkl The reason I am asking about this is that I am wondering what the differences are between master and this branch when running pedestrian routes from here https://github.com/valhalla/valhalla/tree/master/test_requests  I am wondering if there could be a negative impact.  May need to add new tests with areas that have lit tag prevalent and areas where it is not.

Reply:
If you're not changing the `use_lit` request option there shouldn't be any differences at all compared to master. Only when you start with `use_lit > 0` in costing options, there'll be differences. We thought we could avoid doing a RAD testing tbh :innocent: since the impact is very well imaginable IMO, see the graph in @chrstnbwnkl 's OP. Do let us know if you're somewhat concerned, maybe we can talk about it tmrw in the 8:30 review session if you have the time @gknisely ?